### DISCUSSION TITLE
csam black box blocking non adult/child content

### DISCUSSION THREAD

**Discussion by maxclinger** [Haidra-Org/AI-Horde#418] [General]
Does anyone have this issue? Photos or prompts that contain absolutely no sexual content are getting blocked for CSAM content.
Makes AI Horde almost useless

**Comment by db0** [comment]
If you're continuously getting blocked by what you think should be OK content, check that the output is not too "noisy". False positives is a normal thing for this sort of checking unfortunately and can't be helped. But if you're getting constantly blocked you should investigate your input.
